webapp/locales/en.json

```diff
@@ -153,7 +153,8 @@
     },
     "comment": {
       "submit": "Comment",
-      "submitted": "Comment Submitted"
+      "submitted": "Comment Submitted",
+      "updated": "Changes Saved"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-07T14:31:05Z" title="Friday, June 7th 2019, 4:31:05 pm +02:00">Jun 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -145,7 +145,8 @@
     },
     "comment": {
       "submit": "Comment",
-      "submitted": "Comment Submitted"
+      "submitted": "Comment Submitted",
+      "updated": "Comment Updated"
```


@ALau2088 the German translation for `Commented Updated` would be `Kommentar Aktualisiert`. Although I have to admit this sounds a little strange to me. Probably I would use a more generic wording `Changes Saved` <-> `Änderungen gespeichert`.